keras_cv/layers/spatial_pyramid.py

```diff
@@ -31,22 +31,15 @@ class SpatialPyramidPooling(tf.keras.layers.Layer):
     inp = tf.keras.layers.Input((384, 384, 3))
     backbone = tf.keras.applications.EfficientNetB0(input_tensor=inp, include_top=False)
     layer_names = ['block2b_add', 'block3b_add', 'block5c_add', 'top_activation']
```

Comment:
The layer_names is no longer used here anymore.

Reply:
Done.